README.md

```diff
@@ -45,10 +45,10 @@ Fivox provides the following major features:
 Build Fivox from source:
 ~~~
 git clone https://github.com/BlueBrain/Fivox
```

Comment:
it's git clone --recursive, otherwise the submodule for CMake/common is missing. You have not tested it then :)